cmds/stage

```diff
@@ -124,10 +114,33 @@ stage_repository() {
         img_dst_label="${entry[4]}"
         img_dst_mnt="${entry[5]}"
 
+        case "${img_type}" in
```

Comment:
The `case` used to check file extension on `${dst}` which was constructed by `img_dst_name="${img_dst_label}.${img_type}"`, so this `case` statement should remove the leading `.` and can actually remove most of the globing done (except `*vhd.gz` as `ext3.vhd.gz` and `ext3.disk.vhd.gz` are both valid).